sys/zuse.hoon

```diff
@@ -8175,14 +8175,26 @@
   |%
   :: +come:dawn: mine a comet under a star
   ::
+  ::    Randomly generates comet addresses until we find one whose parent is
+  ::    in the list of supplied stars. Errors if any supplied ship
```

Comment:
Can we not pick a random star from the list, and then generate a random number to prepend to it? This way we don't have to retry x amount of times.